test/remote/gateways/remote_vpos_without_key_test.rb

```diff
@@ -0,0 +1,125 @@
+require 'test_helper'
+
+class RemoteVposWithoutKeyTest < Test::Unit::TestCase
```

Comment:
What will be the implications of having two remote test files after this change is made? Will one of them no longer be needed? Genuine curiousity.

Reply:
Please ignore this comment, I totally missed your block of notes above. I only saw what was in the original commit message.